forch/faucetizer.py

```diff
@@ -141,8 +141,9 @@ def _faucetize(self):
                     continue
 
                 port_cfg['native_vlan'] = vid
+                port_cfg['acls_in'] = ['tail_acl']
```

Comment:
this should be an else clause of the if, not something that is assigned and then overwritten

Reply:
Done